rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistStorageProvider.java

```diff
@@ -0,0 +1,8 @@
+package co.rsk.peg.whitelist;
+
+public interface WhitelistStorageProvider {
+
+    void saveLockWhitelist();
```

Comment:
If possible, providing javadoc would be great :)

Reply:
Done!